### ISSUE TITLE
Unicode characters create a 500 error

### ISSUE THREAD

**Issue by benkasminbullock** [metacpan/metacpan-grep-front-end#5]
As a test of the Unicode facility in the grep, I tried the following search:

https://grep.metacpan.org/search?size=20&q=Sprawd%C5%BAmy%2C+czy+odgadywacz+j%C4%99zyk%C3%B3w+pracuje&qd=&qft=

The result should point to the following document:

https://metacpan.org/pod/Lingua::Guess

However, it gave me a 500 error.

**Comment by atoomic** [comment]
Wow this is interesting this is coming from Dancer::CORE Cookie plugin
```
Can't escape \x{017A}, try uri_escape_utf8() instead at .../perl-5.22.1/lib/site_perl/5.22.1/Dancer2/Core/Cookie.pm line 15.
```

code context
```
12     my $self   = shift;
13     my $header = '';
14 
15     my $value = join( '&', map uri_escape($_), $self->value );
16     my $no_httponly = defined( $self->http_only ) && $self->http_only == 0;
17 
18     my @headers = $self->name . '=' . $value;
```

**Comment by toddr** [comment]
It's not a 500 now but now it says it found 1 file but displays nothing?

**Comment by atoomic** [comment]
The 500 issue is fixed by 56fd1c0, but unicode characters are stripped from the query for now, so this is search is at this time not supported.

Would be good to add unicode support in the future.
